EventsExpress/ClientApp/src/components/app/app.js

```diff
@@ -65,8 +67,8 @@ export default class App extends Component {
                         <Route path="/contactUs" component={ContactUsWrapper} />
                         <Route path='/admin/unitsOfMeasuring' component={UnitOfMeasuring} />
                         <Route path='/event/createEvent' component={AddEventWrapper} />
-                        
-
+                        <Route path='/editEvent/:id/' component={DraftEditWrapper} />
+                        <Route path='/Drafts/:page' component={EventDraftListWrapper} />
```

Comment:
This route does not have to have parameter `:page` if you are not storing the page there. Please remove it.